packages/manager/modules/emailpro/src/dashboard/emailpro.controller.js

```diff
@@ -206,6 +206,10 @@ export default /* @ngInject */ function EmailProCtrl(
           loadPtrTooltip($scope.exchange);
           loadPtrv6Tooltip($scope.exchange);
         }
+        if ($scope.exchange.isMXPlan) {
+          const { isZimbra } = await EmailPro.retrieveMxPlan();
+          $scope.isMxPlanMigrated = isZimbra;
+        }
```

Comment:
**suggestion:** `async ... await` requires manual run of digest cycle. Better to switch to conventional promise.